banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py

```diff
@@ -114,7 +114,7 @@ def sync_all_accounts_and_transactions():
 
 	# Update all bank accounts
 	for entry in bank_consents:
```

Comment:
Can't this entire loop be conditional on `Admin().consent_accounts()` succeeding? Everything inside seems relevant for newly fetched accounts only.